examples/lib/stories/effects/effects.dart

```diff
@@ -67,5 +68,14 @@ void addEffectsStories(Dashbook dashbook) {
       'Color Effect',
       (_) => GameWidget(game: ColorEffectGame()),
       codeLink: baseLink('effects/color_effect.dart'),
+    )
+    ..add(
+      'Remove Effect',
+      (_) => GameWidget(game: RemoveEffectGame()),
+      codeLink: baseLink('effects/remove_effect_game.dart'),
+      info: '''
```

Comment:
We have started to move in these info sections to variables in the example class that is it running (it is far from done), so that the info text can be seen easiliy when looking at the code too.

Reply:
Like this:  https://github.com/flame-engine/flame/blob/main/examples/lib/stories/collision_detection/multiple_shapes.dart#L12

Reply:
Ah, that makes sense. I'll do the same here.